docs/08_troubleshooting.md

```diff
@@ -0,0 +1,132 @@
+## Troubleshooting
```

Comment:
This document should probably take a different format.

Instead of a list, each troubleshooting item (the problem) should be a heading. This will enable the Table of Contents to automatically generate something useful, and aid with indexing and discoverability.